axum/CHANGELOG.md

```diff
@@ -37,7 +37,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   
   let app = Router::new()
       .nest("/api", api_router)
-      // `api_fallback` will inherit this fallback
+      // `api_router` will inherit this fallback
```

Comment:
Reasoning: 

- `api_fallback` does not exist here. 
- The `Router` `api_router` inherits the `app_fallback` handler as fallback.